mangaki/mangaki/utils/anidb.py

```diff
@@ -183,6 +181,33 @@ def get_or_update_work(self,
             if title_type == 'synonym':
                 synonyms[lang] = title
 
+        # Handling of staff
+        creators = []
+        staff_map = dict(Role.objects.filter(slug__in=['author', 'director', 'composer']).values_list('slug', 'pk'))
```

Comment:
Arguably, you don't have to bother restricting the queryset to these slugs only.
You could just fetch all `Role`s objects.

Also, let's put a `# FIXME: cache this query`, like `lang_cache` because the contents will rarely change during the lifetime of a deployment.